source/tests/test.ts

```diff
@@ -507,6 +507,17 @@ test('is.dataView', t => {
 	testType(t, 'arrayBuffer');
 });
 
+test('is.directInstanceOf', t => {
+	const error = new Error();
+	const errorSubclass = new ErrorSubclassFixture();
+
+	t.true(m.directInstanceOf(error, Error));
+	t.true(m.directInstanceOf(errorSubclass, ErrorSubclassFixture));
+
+	t.false(m.directInstanceOf(error, ErrorSubclassFixture));
+	t.false(m.directInstanceOf(errorSubclass, Error));
```

Comment:
`t.false(m.directInstanceOf(errorSubclass, Error));` is really the use case here.

Included `t.false(m.directInstanceOf(error, ErrorSubclassFixture));` for completeness.